pravega-cli/src/test/resources/passwd

```diff
@@ -0,0 +1 @@
+admin:353030303a633132666135376233353937356534613430383430373939343839333733616463363433616532363238653930346230333035393666643961316264616661393a3639376330623663396634343864643262663335326463653062613965336439613864306264323839633037626166663563613166333733653631383732353134643961303435613237653130353633633031653364366565316434626534656565636335663666306465663064376165313765646263656638373764396361:*,READ_UPDATE;
```

Comment:
The resource format has changed now, with the merge of [a PR](https://github.com/pravega/pravega/pull/5051/) yesterday in Pravega main.  I suppose that'd not affect the working here as it shall be run against 0.8.x version.

Reply:
This would be run against a 0.8.x version. The resource format change seems to be a 0.9 feature.